### PR TITLE
Add a SQL question regarding Microsoft Access

### PR DESCRIPTION
**Pull request by a2937**
It was a question to ensure any future developers will not seriously consider using Microsoft Access for storing data for their applications. While unlikely, I believe this question is worth adding. 
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

### CODE REVIEW

**Comment by a2937** [comment]
Eslint and prettier are having difficulty locally. They don't give me any output and it's a little unnerving.

File: .prettyignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Ignore artifacts:
+build
+coverage
+
```
Comment (jdwilkin4) [inline .prettyignore:4]: Make sure to fix the file name to `.prettierignore`

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your quiz question submission 👍 
Left a small comment for the file name

File: .prettyignore (hunk 1 above)
Comment (a2937) [reply .prettyignore]: Fixed now.

**Comment by a2937** [comment]
Btw locally I don't see any get any errors regarding the full-quiz. Is this related to checkout or something?

**Review by jdwilkin4 — APPROVED** [review]
@a2937 
I went ahead and removed the lint-style rule from the workflow and it has successfully passed the checks now. 

This PR is good to go 👍

**Pull request merged**